runelite-client/src/main/java/net/runelite/client/plugins/kingdomofmiscellania/KingdomPlugin.java

```diff
@@ -102,6 +106,40 @@ private void processInfobox()
 
 	}
 
+	private void calculateRewards()
+	{
+		ResourceType type = ResourceType.FARM;
```

Comment:
Can you access what the current worker distribution is? It might be in varbits. If so you could show the current profit (in addition to this?) which I think people would like too.

Reply:
I wasn't able to determine the distribution through varbits before but I'm going to check again.

Reply:
You absolutely can, I'll implement this now.

Reply:
Current profit is now calculated. Tested logging in to Miscellania vs. teleporting via Fairy Ring to ensure current amount was being calculated. Also tested changing worker distributions and it seemed to work fine.